apps/ui/src/components/Ui/ChartTime.vue

```diff
@@ -0,0 +1,293 @@
+    areaBottomColor: 'rgba(17, 17, 17, 0.04)'
+  },
+  dark: {
+    lineColor: 'rgba(251, 251, 251, 0.8)',
+    areaTopColor: 'rgba(251, 251, 251, 0.4)',
+    areaBottomColor: 'rgba(251, 251, 251, 0.04)'
+  }
+};
+
+const CHART_COLORS = {
+  light: {
+    textColor: 'rgb(87, 96, 106)',
+    gridLineColor: 'rgba(229, 229, 230, 0.7)',
+    axisLineColor: 'rgb(229, 229, 230)',
+    crosshairLabelBackgroundColor: 'rgb(17, 17, 17)'
+  },
+  dark: {
+    textColor: 'rgb(160, 159, 164)',
+    gridLineColor: 'rgba(47, 46, 51, 0.7)',
+    axisLineColor: 'rgb(47, 46, 51)',
+    crosshairLabelBackgroundColor: 'rgb(251, 251, 251)'
+  }
+};
+
+const OPTIONS = {
+  layout: {
+    background: { color: 'transparent' },
+    attributionLogo: false,
+    fontFamily: 'Calibre',
+    fontSize: 14
+  },
+  grid: {
+    vertLines: { color: 'transparent' },
+    horzLines: { style: LineStyle.Dotted }
+  },
+  crosshair: {
+    vertLine: { color: 'transparent' },
+    horzLine: { color: 'transparent' }
+  },
+  autoSize: true,
+  rightPriceScale: {
+    borderVisible: false,
+    scaleMargins: {
+      top: 0.2,
+      bottom: 0.2
+    }
+  },
+  handleScroll: {
+    vertTouchDrag: false,
+    mouseWheel: false,
+    pressedMouseMove: false
+  },
+  handleScale: {
+    axisPressedMouseMove: {
+      price: false,
+      time: false
+    },
+    mouseWheel: false,
+    pinch: false
+  },
+  timeScale: {
+    timeVisible: true,
+    secondsVisible: false,
+    allowBoldLabels: false,
+    fixLeftEdge: true,
+    fixRightEdge: true,
+    tickMarkMaxCharacterLength: 23,
+    tickMarkFormatter: (time: number, tickMarkType: number) => {
+      const date = new Date(time * 1000);
+
+      if (
+        tickMarkType === TickMarkType.Month ||
+        tickMarkType === TickMarkType.DayOfMonth
+      ) {
+        return date.toLocaleDateString('en-US', {
+          month: 'short',
+          day: 'numeric',
+          timeZone: 'UTC'
+        });
+      }
+
+      return null;
+    }
+  },
+  localization: {
+    timeFormatter: (time: number) => {
+      const date = new Date(time * 1000);
+
+      return date.toLocaleString('en-US', {
+        year: 'numeric',
+        month: 'short',
+        day: 'numeric',
+        hour: '2-digit',
+        minute: '2-digit',
+        hour12: false,
+        timeZone: 'UTC'
+      });
+    }
+  }
+};
+
+const props = defineProps<{
+  series: ChartSeries[];
+}>();
+
+const chartContainer = ref<HTMLElement | null>(null);
+const chart = shallowRef<IChartApi | null>(null);
+const seriesInstances = shallowRef<ISeriesApi<SeriesType>[]>([]);
+
+const { currentTheme } = useTheme();
+
+// Convert UTC timestamp to local timezone offset
+// (effectively makes the chart interpret UTC timestamps as local time)
+// see https://github.com/tradingview/lightweight-charts/blob/7104e9a4fb399f18db7a2868a91b3246014c4324/docs/time-zones.md
+// All further data manipulations should be done in UTC, to avoid double offsets
+const timeToLocale = computed(() => {
```

Comment:
I think it should be called `convertTimeTolocale` so there is some verb as it's a function, but given it's computed that returns a function makes it a bit weird. I feel like just making a normal function and recomputing is not a big deal and makes it less weird.